usr/share/rear/skel/BAREOS/usr/lib/systemd/system/bareos-fd.service

```diff
@@ -0,0 +1,26 @@
+#
```

Comment:
is there a reason for carrying this file in ReaR vs. simply using `COPY_AS_IS` to copy it from the source system`

I really would like to avoid forking Bareos code as that would mean that we have to update it.

look here for a very typical example:
https://github.com/rear/rear/blob/24680de5be94f55ecf8a40a406e8ee13e9a95502/usr/share/rear/conf/default.conf#L2243-L2261

Reply:
Actually there are 2 reasons:
* the systemd service requires some targets not available in Rear: network.target nss-lookup.target time-sync.target
* in Rear the start the bareos-fd in Restore-only mode (parameter -r). This prevents that the server triggers a backup of the Rear system (which would have the additional disadvantage that the last backup would than be the backup of the rear system and the automatic restore would pick the wrong backup)

Reply:
@joergsteffens and I talked about it, ReaR will not ship a fork of upstream systemd files